src/main/java/io/javalin/metrics/JavalinMetrics.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2019 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin.metrics
+
+import io.micrometer.core.instrument.Metrics
+import io.micrometer.core.instrument.Tag
+import io.micrometer.core.instrument.binder.jetty.JettyServerThreadPoolMetrics
+import io.micrometer.core.instrument.binder.jetty.JettyStatisticsMetrics
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry
+import org.eclipse.jetty.server.handler.StatisticsHandler
+import org.eclipse.jetty.util.thread.ThreadPool
+
+class JavalinMetrics private constructor(jettyStatisticsHandler: StatisticsHandler, jettyThreadPool: ThreadPool) {
```

Comment:
Could this be an object instead?

```kotlin
object JavalinMetrics {

    @JvmStatic
    fun init(jettyStatisticsHandler: StatisticsHandler, jettyThreadPool: ThreadPool) {
        ...
    }

}
```

Kotlin's `object` is designed to be used for singletons: https://kotlinlang.org/docs/reference/object-declarations.html#object-declarations.

Reply:
done

thanks for the hint :+1: still getting used to kotlin